src/server/views/section-end-summary.html

```diff
@@ -10,26 +10,20 @@
 {% block content %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      {% if pageHeading %}
-        <h1 class="govuk-heading-l">{{ pageHeading | evaluate }}</h1>
-      {% endif %}
-
       {% if checkAnswers %}
-        {% for section in checkAnswers %}
-          {% if section.title and section.title.text %}
-            <h2 class="govuk-heading-m">{{ section.title.text }}</h2>
-          {% endif %}
-          {% if section.summaryList %}
-            {{ govukSummaryList(section.summaryList) }}
-          {% endif %}
-        {% endfor %}
+        {% if checkAnswers.title and checkAnswers.title.text %}
```

Comment:
I'm still dont think this is correct, if the this condition is false then this page will not have a `h1`

Reply:
I mean, this should never happen. Do you think this should be a breaking error to go to a error page, or a default title?